src/__tests__/integration/internal-writes/internal-write-callee.test.ts

```diff
@@ -279,4 +278,100 @@ describe('Testing internal writes', () => {
       expect((await calleeContract2VM.readState()).cachedValue.state.counter).toEqual(634);
     });
   });
+
+  fdescribe('with internal writes throwing exceptions', () => {
+    beforeAll(async () => {
+      await deployContracts();
+    });
+
+    it('should auto throw on default settings', async () => {
```

Comment:
@jakub-wojciechowski here are new test cases:
1. 'default' settings - should auto throw
2. `evaluationOptions.throwOnInternalWriteError` set to `false` - should not auto throw
3. `evaluationOptions.throwOnInternalWriteError` set to `true` - should auto throw
4. `evaluationOptions.throwOnInternalWriteError` set to `true` but `throwOnError` in `SmartWeave.contracts.write` set to false - should not auto throw
5. `evaluationOptions.throwOnInternalWriteError` set to `false` but `throwOnError` in `SmartWeave.contracts.write` set to true - should throw

Reply:
Looks good, just double checking that default works exactly like `evaluationOptions.throwOnInternalWriteError` explicitly set to `true`, so when
6. `default` but `throwOnError` in `SmartWeave.contracts.write` set to `false` - should not auto throw

Reply:
it should behave like this, but I'll add a separate test case :-)

Reply:
test case added, works as expected.